pandas/tests/io/parser/test_network.py

```diff
@@ -159,7 +159,7 @@ def test_parse_public_s3_bucket_nrows_python(self, tips_df):
             assert not df.empty
             tm.assert_frame_equal(tips_df.iloc[:10], df)
 
-    def test_s3_fails(self):
+    def test_read_s3_fails(self):
         with pytest.raises(IOError):
```

Comment:
can you give a comment on what this test is testing (credentials / invalid path etc)

Reply:
Good point - added